Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM httpd:latest
+
+# Copy configuration file for apache
+COPY ./apache-httpd.conf /usr/local/apache2/conf/httpd.conf
+
+# Expose apache
+EXPOSE 80
+
+# Copy the project files into place.
+COPY public /usr/local/apache2/htdocs/public
```

Comment:
Really? Interesting! I was looking for this myself about two weeks ago but came across nothing decent (also using react router and react). Nice :)